File: src/main/scala/uk/co/ridentbyte/view/response/ClearResponsePane.scala
package uk.co.ridentbyte.view.response

import javafx.scene.layout._
import javafx.scene.paint.Color

case class ClearResponsePane() extends BorderPane {
  getStyleClass.addAll("dashed-border", "round-border")
  setBorder(new Border(new BorderStroke(Color.LIGHTGREY, BorderStrokeStyle.DASHED, CornerRadii.EMPTY, BorderWidths.DEFAULT)))
}

File: src/main/java/uk/co/ridentbyte/view/response/ClearResponsePane.java
package uk.co.ridentbyte.view.response;

import javafx.scene.control.Label;
import javafx.scene.layout.*;
import javafx.scene.paint.Color;

public class ClearResponsePane extends BorderPane {
    public ClearResponsePane() {
        var prompt = new Label("Welcome!");
        prompt.getStyleClass().addAll("cardinal-font-console", "grey-text");
        this.getStyleClass().addAll("dashed-border", "round-border");
        this.setCenter(prompt);
        this.setBorder(
                new Border(
                        new BorderStroke(
                                Color.LIGHTGREY,
                                BorderStrokeStyle.DASHED,
                                CornerRadii.EMPTY,
                                BorderWidths.DEFAULT
                        )
                )
        );
    }
}
